apps/web/lib/features/team-members-kanban-view.tsx

```diff
@@ -163,7 +225,7 @@ export const KanbanView = ({ kanbanBoardTasks }: { kanbanBoardTasks: IKanban })
 			destination
 		});
 
-		setItems(data.kanbanBoard);
+		// setItems(data.kanbanBoard);
```

Comment:
@Anishali2 why it was needed before and not needed now? Why you keep it commented?

Reply:
It's just used for testing purpose and now i don't need that. :)